bin/cheritest/cheritest.c

```diff
@@ -2192,17 +2228,14 @@ cheritest_run_test(const struct cheri_test *ctp)
 		    "unwind");
 		goto fail;
 	}
-#ifdef __mips__
-	if (ctp->ct_flags & CT_FLAG_MIPS_EXCCODE) {
-		mips_exccode = (ccsp->ccs_mips_cause & MIPS_CR_EXC_CODE) >>
```

Comment:
Personally not particularly keen on declaring all variables at the start of the function since this can result in compilation failures such as the one here. But I guess that's what the style guide requires.